src/Application.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TokenSniffer;
+
+use Exception;
+use Symfony\Component\Process\Process;
+
+class Application
+{
+    private const PARAM_EXCLUDE_FILE = 'exclude-file';
+    private const PARAM_EXCLUDE = 'exclude';
```

Comment:
pokud exclude neumí rovnou pracovat adresářema, tak bych řekl, že bys měl explicitně uvést, čeho se exclude týká, tj. třeba `exlucde-file`, `exclude-dir`. jinak asi není problém použít jen `exclude` a za běhu si zkontrolovat, jestli to je file nebo folder.